src/views/3-Modeling/Modeling.vue

```diff
@@ -9,7 +9,7 @@
       <div class="sticky">
         <div id="bees-container">
           <div id="progress-container">
-          <p class="progress"> </p>
+            <p class="progress" />
```

Comment:
not really related to this PR but I was having issues with this `<p>` and it wanting to have a super short line height with a white background. my normal fixes did not seem to work?